### PR TITLE
Add Thin and Fat ID Token Content

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Add content for thin/fat ID tokens
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-371407](https://oktainc.atlassian.net/browse/OKTA-371407)

### CODE REVIEW

**Comment by christinechoy-okta** [comment]
Andrea Skouras and Waldemar Rosenfeld have reviewed and approved.

File: packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md [hunk 1]
```diff
@@ -58,15 +58,26 @@ We use the same terms as the [OpenID Connect](https://openid.net/specs/openid-co
...
 
 The two biggest security benefits of OAuth are using tokens instead of passing credentials and restricting the scope of tokens.
-Both of these measures go a long way toward mitigating the impact of a security compromise.
+Both of these measures go a long way toward mitigating the impact of a security compromise:
 
-* Sending usernames and passwords around is like putting all of your eggs in one basket. By using credentials to obtain a token,
-    if someone gains access to the token, they may be able to use it for a short time, but they haven't compromised the user's identity.
-* Controlling what a token is entitled to access further limits damage in case of compromise. For example, scoping a token for shoppers
-    on a web site, and not allowing them access to change prices, provides significant mitigation.
+* Sending usernames and passwords around is like putting all of your eggs in one basket. By using credentials to obtain a token,if someone gains access to the token, they may be able to use it for a short time, but they haven't compromised the user's identity.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:63]:
```suggestion
* Sending usernames and passwords around is like putting all of your eggs in one basket. By using credentials to obtain a token, if someone gains access to the token, they may be able to use it for a short time, but they haven't compromised the user's identity.
```

File: packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md [hunk 2]
```diff
@@ -58,15 +58,26 @@ We use the same terms as the [OpenID Connect](https://openid.net/specs/openid-co
...
+* Controlling what a token is entitled to access further limits damage in case of compromise. For example, scoping a token for shoppers on a web site, and not allowing them access to change prices, provides significant mitigation.
 
 Okta helps you manage ID Tokens (OpenID Connect) and Access Tokens (OAuth 2.0).
 
+An ID token is used to pass along user profile information, such as first name, last name, and email. OpenID Connect uses the  concepts of Thin ID Token and Fat ID Token, where:
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:68]:
```suggestion
An ID token is used to pass along user profile information, such as first name, last name, and email. OpenID Connect uses the concepts of Thin ID Token and Fat ID Token, where:
```

Comment (jakubvul-okta) [reply packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:68]: @susanharper-okta please chime in, but I feel like we don't capitalize "access token" or "ID token", so "fat" and "thin" should be all lowercase here.

File: packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md [hunk 3]
```diff
@@ -58,15 +58,26 @@ We use the same terms as the [OpenID Connect](https://openid.net/specs/openid-co
...
+* A returned ID Token is considered a Fat ID Token that carries all the profile information.
+
+A Thin ID Token contains base claims (information embedded in a token) and some scope dependent claims. Profile attributes and groups aren't returned. Some examples of when both the ID Token and Access Token are returned are as follows:
+
+* Implicit flow, where `response_type=id_token+token`
+* Authorization code flow, where the `openid` scope is passed in the authorize request
+* Resource owner password flow, where the `openid` scope is passed in the token request
+
+A Fat ID Token returns all user claims, which are all the **profile attributes** and **groups**, if **profile scope** and **groups scope** are passed. When an ID Token is returned without the Access Token, for example, in an implicit flow where `response_type=id_token`, it is considered to be a Fat Token and should contain all the **profile attributes** and **groups**, if **profile scope** and **groups scope** are passed.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:79]: We only use **bold** for clickable UI elements.

File: packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md (hunk 2 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:68]: yup: thin ID token and fat ID token

File: packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md [hunk 4]
```diff
@@ -58,15 +58,26 @@ We use the same terms as the [OpenID Connect](https://openid.net/specs/openid-co
...
-    on a web site, and not allowing them access to change prices, provides significant mitigation.
+* Sending usernames and passwords around is like putting all of your eggs in one basket. By using credentials to obtain a token,if someone gains access to the token, they may be able to use it for a short time, but they haven't compromised the user's identity.
+* Controlling what a token is entitled to access further limits damage in case of compromise. For example, scoping a token for shoppers on a web site, and not allowing them access to change prices, provides significant mitigation.
 
 Okta helps you manage ID Tokens (OpenID Connect) and Access Tokens (OAuth 2.0).
 
+An ID token is used to pass along user profile information, such as first name, last name, and email. OpenID Connect uses the  concepts of Thin ID Token and Fat ID Token, where:
+
+* A returned ID Token and Access Token is considered a Thin ID Token that carries minimal profile information. If you want to retrieve the rest of the information, you'll need to call Okta's `/userinfo` endpoint using the Access Token you receive.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:70]: ID token, access token, thin ID token

avoid future tense

access token THAT you receive.

File: packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md [hunk 5]
```diff
@@ -58,15 +58,26 @@ We use the same terms as the [OpenID Connect](https://openid.net/specs/openid-co
...
+* A returned ID Token is considered a Fat ID Token that carries all the profile information.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:71]: ID token, fat ID token

File: packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md [hunk 6]
```diff
@@ -58,15 +58,26 @@ We use the same terms as the [OpenID Connect](https://openid.net/specs/openid-co
...
+
+A Thin ID Token contains base claims (information embedded in a token) and some scope dependent claims. Profile attributes and groups aren't returned. Some examples of when both the ID Token and Access Token are returned are as follows:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:73]: thin ID token, ID token, access token

File: packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md [hunk 7]
```diff
@@ -58,15 +58,26 @@ We use the same terms as the [OpenID Connect](https://openid.net/specs/openid-co
...
+
+* Implicit flow, where `response_type=id_token+token`
+* Authorization code flow, where the `openid` scope is passed in the authorize request
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:76]: Authorization Code flow

File: packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md [hunk 8]
```diff
@@ -58,15 +58,26 @@ We use the same terms as the [OpenID Connect](https://openid.net/specs/openid-co
...
+* Resource owner password flow, where the `openid` scope is passed in the token request
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:77]: Resource Owner Password flow

**Review by susanharper-okta — COMMENTED** [review]
couple comments

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**